ui/apps/platform/src/Containers/Vulnerabilities/VirtualMachineCves/Overview/VirtualMachinesCvesTable.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from 'react';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import useRestQuery from 'hooks/useRestQuery';
+import { listVirtualMachines } from 'services/VirtualMachineService';
+import { getTableUIState } from 'utils/getTableUIState';
+import TbodyUnified from 'Components/TableStateTemplates/TbodyUnified';
+import { Link } from 'react-router-dom-v5-compat';
+
+function VirtualMachinesCvesTable() {
+    const fetchVirtualMachines = useCallback(() => listVirtualMachines(), []);
+
+    const { data, isLoading, error } = useRestQuery(fetchVirtualMachines);
+
+    const tableState = getTableUIState({
+        isLoading,
+        data: data ?? [],
+        error,
+        searchFilter: {},
+    });
+
+    return (
+        <Table
+            borders={tableState.type === 'COMPLETE'}
+            variant="compact"
+            aria-live="polite"
+            aria-busy={false}
```

Comment:
Hey, can you also add this to office hours.

I feel like maybe these `aria` props were not as clear cut in semantics as they had seemed.